src/test-helpers.tsx

```diff
@@ -944,3 +945,7 @@ export const setMockFetchResponseJSON = (
     },
   });
 };
+
+export const getInstance = <ComponentType extends React.Component>(
```

Comment:
I'm kind of meh about it because it doesn't lessen the calling code by much. I feel like it makes the one-liner type assertions that were in parentheses a little easier to read, though.